cli/args/flags.rs

```diff
@@ -1585,6 +1586,14 @@ fn help_subcommand(app: &Command) -> Command {
     }))
 }
 
+fn add_dev_arg() -> Arg {
+  Arg::new("dev")
```

Comment:
Is there room for a -D? All other package managers has a `deno i -D package` syntax, and frankly the `npm:` is already a divergence from the pack.

Reply:
Oh whoops I actually intended to add that, I just forgot to do it here. Thanks for catching that!